kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```
@@ -621,7 +621,7 @@ netWifiToolTipWirelessModeAdhoc=Configure the wireless interface in adhoc mode.
 netWifiToolTipNetworkName=In <i>Access Point</i> mode, enter the SSID that will identify this wireless network.<br><br>In <i>Station</i> mode, enter the SSID of the wireless network or click the button to select wireless network in range. The button may be disabled for some systems. 
 netWifiToolTipRadioMode=In <i>Access Point</i> mode, select the appropriate speed rating for this device.<br><br>* 802.11a - 54 MBps, 50 feet or 450 MBps,  using 5Ghz channels (up to 32) <br>* 802.11b - 11 Mbps, 150 feet<br>* 802.11g - 54 MBps, 50 feet<br>* 802.11n - 300 MBps, 175 feet<br>* 802.11ac - 1000 MBps, 50 feet<br><br>Note that real-world speeds are more modest. Likewise, range varies depending on device configuration and placement.
 netWifiToolTipBand=Select the frequency band to use for this wireless interface.<br><br><ul><li>2.4 GHz band provides the most coverage but transmits data at slower speeds.</li><li>5 GHz band provides less coverage but transmits data at faster speeds.</li></ul>Wireless range decreases with higher frequencies because higher frequencies cannot penetrate solid objects, such as walls and floors. However, higher frequencies allow data to be transmitted faster than lower frequencies.
-netWifiToolTipSecurity=Select the appropriate security protocol for this wireless network.<br>WPA2 is the most secure encryption method available for wireless networks - we recommend using WPA2 with the CCMP cipher whenever possible. WPA2 with CCMP is the only option permitted for high throughput 802.11n transmissions.<br>If you need to accommodate legacy devices with an SSID, enable WPA encryption with the TKIP cipher. 
+netWifiToolTipSecurity=Select the appropriate security protocol for this wireless network.<br>WPA2 is the most secure encryption method available for wireless networks - we recommend using WPA2 with the CCMP cipher whenever possible. WPA2 with CCMP is the only option permitted for high throughput 802.11n transmissions.<br>If you need to accommodate legacy devices with an SSID, enable WPA encryption with the TKIP cipher.<br><br>The WPA3-SAE security protocol is experimental. Please check if the device, driver and firmware are compliant with the new standard. 
```

I would rephrase this as follows. Let me know what you think

```suggestion
netWifiToolTipSecurity=Select the appropriate security protocol for this wireless network.<br>WPA2 is the most secure encryption method available for wireless networks - we recommend using WPA2 with the CCMP cipher whenever possible. WPA2 with CCMP is the only option permitted for high throughput 802.11n transmissions.<br>If you need to accommodate legacy devices with an SSID, enable WPA encryption with the TKIP cipher.<br><br>The WPA3-SAE security protocol support is experimental. Please check whether the new standard is supported by the Wifi module device, driver and firmware. 
```